### PR TITLE
Configure Harbor with HTTPS Access

### PR DESCRIPTION
**Pull request by xiahaoshawn**
add cert directory and nginx.https.conf file to Deploy/config/nginx/

### CODE REVIEW

File: Deploy/config/nginx/nginx.https.conf [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      proxy_set_header X-Real-IP $remote_addr;
+      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+      proxy_set_header X-Forwarded-Proto $scheme;
+      proxy_buffering off;
+      proxy_request_buffering off;
+    }
+  }
+    server {
+      listen 80;
+      server_name harbordomian.com;
```
Comment (reasonerjt) [inline Deploy/config/nginx/nginx.https.conf]: harbordomain.com

File: Deploy/config/nginx/nginx.https.conf [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  }
+
+  upstream ui {
+    server ui:80;
+  }
+
+
+  server {
+    listen 443 ssl;
+    server_name harbordomian.com;
```
Comment (reasonerjt) [inline Deploy/config/nginx/nginx.https.conf]: there are a few typo, you should update "harbordomian" to "harbordomain"

File: Deploy/config/nginx/nginx.https.conf [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    # SSL
+    ssl_certificate /etc/nginx/conf.d/harbordomian.crt;
```
Comment (reasonerjt) [inline Deploy/config/nginx/nginx.https.conf]: is it possible put the certs to /etc/nginx/cert?

Comment (xiahaoshawn) [inline Deploy/config/nginx/nginx.https.conf]: it's ok and docker-compose.yml also need to change to /etc/nginx/cert

**Comment by xiahaoshawn** [comment]
docker-compose.yml need  change proxy volume and port sections
  volumes:
    - ./config/nginx/nginx.https.conf:/etc/nginx/nginx.conf
    - ./config/nginx/cert:/etc/nginx/cert
  ports:
    - 80:80
    - 443:443

**Comment by xiaods** [comment]
@xiahaoshawn we need update a doc to guide user.

**Comment by cmingxu** [comment]
I am thinking why not put both registry and harbor behind a nginx proxy with https access config.

**Pull request merged**